src/components/color-picker/color-picker.tsx

```diff
@@ -482,7 +481,6 @@ export class ColorPicker implements InteractiveComponent, LoadableComponent {
     }
 
     input.value = inputValue;
-    internalInput.value = inputValue;
```

Comment:
@eriklharper do you remember why you were setting the value for both the internal input and the `calcite-input`? None of the e2e tests are failing :sweat_smile:

Reply:
Nevermind sorry Erik, I was git blaming the wrong line. It's from #1410

@driskull was I not supposed to remove the whole event payload? Or I guess I could go into the shadowRoot...

Reply:
@eriklharper do you know if the whole event payload is deprecated or just the value?

I would say whole thing if a user can still get all the information.

Reply:
I'm not sure why we were exposing the native input event.

Reply:
> @eriklharper do you know if the whole event payload is deprecated or just the value?

I think the whole event payload as long as the values sent to the listener can all be retrieved by the event target object?

Reply:
It's bad practices, but in order to remove the whole event payload I would need to go into  `calcite-input`'s shadowRoot to set the internal input's value. When Franco gets back I can ask him why we needed to do that and if it can be removed. 

The alternative is not setting the value on the internal input (what this PR is currently doing), and hoping it doesn't break anything lol.

Any preferences @driskull / @eriklharper ?

Reply:
@eriklharper do you know why its setting it like this? 

@benelan lets wait for franco on this one. For now, lets leave it alone and add a note to revisit via a comment with `@deprecated`